Mathlib/Topology/Algebra/Module/Multilinear/Basic.lean

```diff
@@ -299,19 +299,56 @@ theorem _root_.ContinuousLinearMap.compContinuousMultilinearMap_coe (g : M₂ 
   ext m
   rfl
 
+/-- `ContinuousMultilinearMap.prod` as an `Equiv`. -/
+@[simps apply symm_apply_fst symm_apply_snd, simps (config := .lemmasOnly) symm_apply]
+def prodEquiv :
+    (ContinuousMultilinearMap R M₁ M₂ × ContinuousMultilinearMap R M₁ M₃) ≃
```

Comment:
Should this be a linear equiv?

Reply:
A `LinearEquiv` needs more typeclass assumptions, e.g., `ContinuousAdd` and `ContinuousSMul`.

Reply:
Weird, I'm surpised that `ContinuousMultilinearMap` is legal without those assumptions! Either way, that's a convincing enough reason to have the `Equiv`.